python/mxnet/optimizer.py

```diff
@@ -356,6 +358,52 @@ def update(self, index, weight, grad, state):
                        lr=lr, wd=wd, **kwargs)
 
 @register
+class MultiPrecisionSGD(Optimizer):
```

Comment:
I'm actually fine with it being part of SGD, although this would probably be better if all of the optimizers had a similar mode.